elasticsearch/elasticsearch.d.ts

```diff
@@ -331,6 +333,11 @@ declare module Elasticsearch {
         refresh?: boolean;
     }
 
+    export interface DeleteByQueryParams extends GenericParams {
+        index: string;
```

Comment:
Yeah, you are right. Those are the same parameters as for the search query. https://www.elastic.co/guide/en/elasticsearch/client/javascript-api/current/api-reference.html#api-search